RoddyCore/src/de/dkfz/roddy/plugins/LibrariesFactory.groovy

```diff
@@ -308,48 +308,47 @@ public class LibrariesFactory extends Initializable {
         return file.exists() && file.isDirectory() && file.canRead() && file.canExecute()
     }
 
-    static PluginType determinePluginType(File directory, Map<String, List<String>> mapOfErrors = [:]) {
-        logger.postRareInfo("  Parsing plugin folder: ${directory}");
+    static PluginType determinePluginType(File file, Map<String, List<String>> mapOfErrors = [:]) {
```

Comment:
Why did you rename it to file? It is supposed to be a directory (see first check below)

Reply:
You're right. Changed it back.